cmd/errtrace/testdata/golden/noop.go

```diff
@@ -2,8 +2,32 @@
 
 package foo
 
+import "errors"
+
 // This file should not be changed.
 
 func success() error {
 	return nil
 }
+
+func failure() error {
```

Comment:
suggest mixing non-skipped error returns with skipped returns in the same file to validate that the position of the return matches what we expect